Resources/Prototypes/Entities/Clothing/Back/duffel.yml

```diff
@@ -14,6 +14,7 @@
     walkModifier: 1
     sprintModifier: 0.9
   - type: HeldSpeedModifier
+    halfSpeedModifier: true
```

Comment:
not necessary, given that this is the standard value in the component
```suggestion
```

Reply:
oh i was about to set false here, thanks for notice